### DISCUSSION TITLE
error generating NFAuthentication  key

### DISCUSSION THREAD

**Discussion by marcoriccine** [CastagnaIT/plugin.video.netflix#1631] [Q&A]
I have done this process again, but this time  something goes wrong.
Very old and slow linux mint device, first time chrome was too old, no data has returned.
updated the browser and this is what i have:

Press "ENTER" key to accept the disclaimer and start, or "CTRL+C" to cancel
Browser startup... please wait
Establish connection with the browser... please wait
An error is occurred:
Handshake status 403 Forbidden

I will try on windows device... but il'd like to know why!!
thank's Marco

**Comment by Mannshoch** [comment]
In my case: https://github.com/CastagnaIT/plugin.video.netflix/issues/1629
Firefox did not work. Only Chromium.
